pkg/skaffold/deploy/kustomize/kustomize.go

```diff
@@ -189,7 +189,7 @@ func (k *Deployer) renderManifests(ctx context.Context, out io.Writer, builds []
 		return nil, nil
 	}
```

Comment:
It would be useful to have similar tracing in the kustomize deployer.

Reply:
Done